future/src/ct/ct_parser_text.cc

```diff
@@ -20,6 +20,7 @@
  */
 
 #include "ct_parser.h"
+#include "ct_logging.h"
```

Comment:
@nutianlab This may be culprit, `ct_logging.h` is included before `fmt/fmt.h` @txe may have just fixed it with #918 but I am going to test it on clang now